compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1050,6 +1050,12 @@ error.array.index.out.of.range=\
 error.sealed.array.type.can.not.infer.size=\
   invalid usage of sealed type: can not infer array size
 
+error.invalid.key.func.return.type=\
+  invalid array sort key function return type: type ''{0}'' is not an ordered type
+
+error.invalid.sort.array.member.type=\
+  invalid member type of the array to sort: type ''{0}'' is not an ordered type
```

Comment:
Just wondering whether it'd be better to avoid saying "array", considering that this applies to tuples as well? @MaryamZi 

Also, the first  "type" in the message after the colon seems redundant.

Reply:
@pubudu91 @MaryamZi any suggestions on what it should be?